### PR TITLE
52 feature pagination UI condensed

### PR DESCRIPTION
**Pull request by LucRoy**
Change the pagination component to be more friendly to large numbers of replications.

PP Options [100(default), 50, 25, 10]
Screenshot 2023-05-31 at 3 48 09 PM

https://github.com/application-research/delta-ui/assets/21270122/e763e680-61d0-4ea2-98e3-10754a7b2e63

### CODE REVIEW

File: components/apps/ddm/scenes/Replications.tsx [hunk 1]
```diff
@@ -167,26 +168,40 @@ export default function Replications(props: {
...
-function PageIndex(props: { offset: number, onChangeOffset: (number) => void, limit: number, total: number }) {
+function PageIndex(props: { offset: number, onChangeOffset: (number) => void, onSearchLimit: (number) => void, limit: number, total: number }) {
   const pageCount = Math.ceil(props.total / props.limit);
-  const currPage = Math.floor(props.offset / props.limit);
+  let currPage = Math.floor(props.offset / props.limit);
 
-  const itemCount = Math.min(props.limit, props.total - props.offset);
-  const firstItem = props.offset + 1;
-  const lastItem = props.offset + itemCount;
+  const limitOptions = [
```
Comment (jcace) [inline components/apps/ddm/scenes/Replications.tsx:192]: in Boost, the user's selection actually get saved in LocalStorage, so they only have to set it once and then the limit will persist until its changed again. Could we do that in DDM UI as well? 

Screenshot 2023-05-31 at 4 06 08 PM

Comment (LucRoy) [reply components/apps/ddm/scenes/Replications.tsx]: I've added the localStorage in latest. Good catch 👍

File: components/apps/ddm/DDM.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ export default function DDM(props) {
     setCookie('ddm-address', ddmAddress);
   };
 
-  const [getReplicationsConfig, setGetReplicationsConfig] = React.useState({ limit: 100 } as GetReplicationsConfig);
+  const [getReplicationsConfig, setGetReplicationsConfig] = React.useState({ limit: parseInt(localStorage.getItem("settings.replications.limit")) || 100 } as GetReplicationsConfig);
```
Comment (jcace) [inline components/apps/ddm/DDM.tsx:48]: nit: I think we should default it to `10` as the smallest option so it's less overwhelming and the user can scale up as they need - that's how Boost does it

**Review by jcace — APPROVED** [review]

File: components/apps/ddm/DDM.tsx (hunk 2 above)
Comment (LucRoy) [reply components/apps/ddm/DDM.tsx]: default should now be 10 👍

**Review by elijaharita — APPROVED** [review]
beautiful!

**Pull request merged**